docs/deployment/tracing.md

```diff
@@ -44,7 +44,7 @@ As you can see, we retrieved the list of vets in our database in just under 40ms
 
 In order to enable tracing we'll take a look at the above example to show you what to set up. At a high level, you'll need:
 
-1. wasmCloud [v0.55.1](https://github.com/wasmCloud/wasmcloud-otp/releases/tag/v0.55.1) or later with the following environment variables:
+1. wasmCloud [v0.78.0](https://github.com/wasmCloud/wasmCloud/releases) or later with the following environment variables:
```

Comment:
```suggestion
1. wasmCloud [v0.78.0](https://github.com/wasmCloud/wasmCloud/releases/tag/v0.78.0) or later with the following environment variables:
```

I assume this should have been a direct link to `0.78.0`?

Reply:
I figured linking to the general release page would make this link more perma-y. I don't want to have to update this again

And now that I mention it, I think removing the version entirely makes more sense. We've had tracing support for like 18 months so no need to specify a version

Reply:
```suggest
1. wasmCloud [v0.78.0](https://github.com/wasmCloud/wasmCloud/releases) or later with the following environment variables:
1. wasmCloud [v0.78.0 or later](https://github.com/wasmCloud/wasmCloud/releases) with the following environment variables:
```